permissions/plugin-azure-ad.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "azure-ad"
+paths:
+- "org/jenkins-ci/plugins/azure-ad"
+developers:
+- "chenkennt"
+- "arieshout"
+- "raphaelyu"
```

Comment:
Just to confirm, this is the correct user? All others are `@microsoft.com`.

Reply:
Exactly, this is the main contributor of this plugin